libursa/src/bn/openssl.rs

```diff
@@ -190,6 +190,7 @@ impl BigNumber {
         Ok(bn)
     }
 
+    // TODO: There should be a mod_sqr using underlying math library's square modulo since squaring is faster.
```

Comment:
We can just use mod_mul where you multiply by itself.

Reply:
My comment says about why squaring is preferred.